docs/markdown/Unit-tests.md

```diff
@@ -27,15 +26,31 @@ test('envvar test', exe2, env : ['key1=value1', 'key2=value2'])
 
 Note how you need to specify multiple values as an array.
 
-Coverage
---
+### MALLOC_PERTURB_
+
+By default, environment variable
+[`MALLOC_PERTURB_`](http://man7.org/linux/man-pages/man3/mallopt.3.html)
+is set to a random value between 1..255. This can help find memory
+leaks on configurations using glibc, including with non-GCC compilers.
+However, this can have a performance impact, and may fail a test even on
+external libraries whose internals are out of the user's control.
+If a test() unexpectedly fails, try disabling this feature by adding
+
+```meson
+nomalloc = environment({'MALLOC_PERTURB_': '0'})
+
+test(..., env: nomalloc, ...)
+```
+
+or setting environment variable `MALLOC_PERTURB_=0` to disable for all tests.
```

Comment:
This is duplicated text. This should instead say that `MALLOC_PERTURB_` behaviour is the same as in the other page and have a link to it instead.